elkscmd/ash/autocomplete.c

```diff
@@ -20,7 +21,7 @@ void completion(const char *buf, linenoiseCompletions *lc)
 	DIR *fp;
 	struct dirent *dp;
 	int count;
-	char dir[128];
+	char dir[NAME_MAX];
```

Comment:
It seems that `dir[]` gets its input from sources other than `readdir`, and copying to it isn't yet checked. I would suggest we keep `dir[]` at PATH_MAX for now (along with an explanatory comment) until we can show or know it can be made smaller.

Reply:
fixed